docs/reference/operator_parameters.md

```diff
@@ -294,6 +294,11 @@ In the CRD-based configuration they are grouped under the `load_balancer` key.
   cluster.  Can be overridden by individual cluster settings. The default is
   `false`.
 
+* **custom_service_annotations**
+  when load balancing is enabled, LoadBalancer service is created and
+  this parameter takes service annotations that are applied to service.
+  Optional.
```

Comment:
is this applied to both replica and master LBs if they are enabled ?

Reply:
yes, that is correct, to both replica and master, but only LB type.